README.md

```diff
@@ -13,7 +13,7 @@
 
 ## Live Demo (if available)
 
-[Live Demo Link](https://livedemo.com)
+https://yetemegn-telaye.github.io/Blockchain-Learning-website/
 
 
 ## Getting Started
```

Comment:
Kindly update your Readme file as per this project. Consider updating your description, some of the sections you have left empty like Prerequisites, and Setup should be updated according to this project. You can remove some sections which are not relevant to this project like Run tests etc. Remember, it is super important to have a good project but a great README file, since anyone who is visiting your project here on GitHub sees the README file first then your code.
- Also, kindly link the profile of Cindy Shin in the acknowledgment section.